data/remnant/remnant ships.txt

```diff
@@ -1183,33 +1188,30 @@ ship "Peregrine"
 		licenses
 			"Remnant Capital"
 		"cost" 41627000
-		"shields" 16400
-		"hull" 10600
+		"shields" 15400
+		"hull" 8600
```

Comment:
```suggestion
		"shields" 16400
		"hull" 10600
```
The Peregrine's total HP of 27000 is not an excessively large step up from the Ibis' 20600, considering the technological advancement the Peregrine represents and the Ibis' role as a carrier. So, I don't think the HP needs to be reduced.